### PR TITLE
Zip the binaries instead of using UPX

### PR DESCRIPTION
**Pull request by datho7561**
Signed-off-by: David Thompson <davthomp@redhat.com>

### CODE REVIEW

File: NativeImage.jenkins [hunk 1]
```diff
@@ -153,11 +152,11 @@ pipeline {
           }
           sh "mkdir ${vscodeXmlVersion}-${env.BUILD_NUMBER}"
           sh """
-            cp lemminx-linux ${vscodeXmlVersion}-${env.BUILD_NUMBER}/lemminx-linux
+            cp lemminx-linux ${vscodeXmlVersion}-${env.BUILD_NUMBER}/lemminx-linux.zip
```
Comment (fbricon) [inline NativeImage.jenkins:155]: cp lemminx-linux.zip

File: NativeImage.jenkins [hunk 2]
```diff
@@ -153,11 +152,11 @@ pipeline {
...
             cp lemminx-linux.sha256 ${vscodeXmlVersion}-${env.BUILD_NUMBER}/lemminx-linux.sha256
-            cp lemminx-win32.exe ${vscodeXmlVersion}-${env.BUILD_NUMBER}/lemminx-win32.exe
+            cp lemminx-win32.exe ${vscodeXmlVersion}-${env.BUILD_NUMBER}/lemminx-win32.zip
```
Comment (fbricon) [inline NativeImage.jenkins:157]: cp lemminx-win32.zip

File: NativeImage.jenkins [hunk 3]
```diff
@@ -132,12 +132,11 @@ pipeline {
         unstash name: 'lemminx-win32.exe'
         unstash name: 'lemminx-osx-x86_64'
 
-        // Run UPX to reduce binary size
```
Comment (fbricon) [inline NativeImage.jenkins:135]: rename stage to 'Zip and upload'

**Review by rgrunber — APPROVED** [review]
LGTM.. after fixing what Fred mentioned..

**Pull request merged**